### PR TITLE
Added Spanish translation

### PR DESCRIPTION
**Pull request by AngelGryph**
Spanish translation by alaisFcZ.  I'm only uploading it, I do not speak a word of Spanish myself.

### CODE REVIEW

File: ntotsc/base/cre/nostring/ntkrotan.cre
Comment (Roberciiik) [inline ntotsc/base/cre/nostring/ntkrotan.cre:1]: Why are you chaning ntkrotan.CRE file?

**Comment by Roberciiik** [comment]
Hello @AngelGryph ! Could you please update the README file with the authors of the translation? Thank you!

**Comment by 11jo** [comment]
Hello @Roberciiik,

I just receive a message AlaisFcZ from the autor of the Spanish translation asking me if it's possible to merge it.

Do you want me to send a pull request with the author name to facilitate things for him ?

**Comment by Roberciiik** [comment]
> Hello @Roberciiik,
> 
> I just receive a message from AlaisFcZ the autor of the Spanish translation asking if it's possible to merge it officially.
> 
> Do you want me to send you a pull request with the author name in the readme to facilitate things for him ?

@11jo Yes, please, if you would like to update this or create a new PR, then I will be happy to merge it. I am not sure why @AngelGryph updated CRE file. Probably not finished Castilian language should be removed as it is northern version of Spanish.

**Comment by 11jo** [comment]
aaa

For the cnkrotan.cre, I did some comparaison and the change seem very justified.

**Comment by Roberciiik** [comment]
I will merge that one, and you could remove ogg files (as install script will use English if local is not provided). And also add proper README recognition.

**Pull request closed without merging**